ocaml/idl/datamodel.ml

```diff
@@ -3946,7 +3950,11 @@ let host_metrics_memory =
   let field = field ~ty:Int in
   [
     field ~qualifier:DynamicRO "total" "Total host memory (bytes)" ~doc_tags:[Memory];
-    field ~qualifier:DynamicRO ~internal_deprecated_since:rel_midnight_ride "free" "Free host memory (bytes)" ~doc_tags:[Memory];
+    field "free" "Free host memory (bytes)"
+      ~lifecycle:[ Removed, rel_tampa, "Disabled in favour of RRD - see xcp-rrdd"]
```

Comment:
The `~internal_deprecated_since` argument below was meant to be replaced by `~lifecycle`. To avoid confusion, it's better to remove it, and add a lifecycle entry:

    ~lifecycle:[ Deprecated, rel_midnight_ride, "..."; Removed, rel_tampa, "..."]